CMakeLists.txt

```diff
@@ -15,7 +19,11 @@ if(NOT CMAKE_OSX_DEPLOYMENT_TARGET)
         FORCE)
 endif()
 
+if(APPLE)
+project(transmission LANGUAGES C CXX Swift)
```

Comment:
This duplication doesn't seem necessary. Just enable the additional language in macosx/

```cmake
# macosx/CMakeLists.txt
enable_language(Swift)
```

Reply:
done in 8d80cbb1cd12ccb256b9846f65415dc914309422